### ISSUE TITLE
Can't import Flickr photos to taxon page

### ISSUE THREAD

**Issue by tiwane** [inaturalist/inaturalist#3611]
From https://forum.inaturalist.org/t/cannot-add-taxon-images/38163 I can't replicate

1. Go to https://www.inaturalist.org/taxa/544918-Archimantis-quinquelobata and Edit photos
2. Choose Flickr as your photo source
3. [This photo](https://flic.kr/p/2o1UoWq) of mine should show up. Drag it over and drop it. 
4. Everything looks good, click Save Photos.
5. Modal disappears but the photo from Flickr is never added.

**Comment by csiro-chris-m** [comment]
This issue has not been resolved. I still cannot add any images to a taxon. I tried in Chrome and in Firefox and I’ve tested it in the V2 API with the same effect, no images added. Can this be escalated?

**Comment by Murray-Dawson** [comment]
Flickr is an important part of my workflow too - I just checked and the Flickr feed within iNat is still working for me. Maybe try logging out and then back into your Flickr account in a separate browser tab??? Mine is a paid version of Flickr - have you reached the 1,000 image/video limit if yours is the free Flickr account??

**Comment by csiro-chris-m** [comment]
Hi

I appreciate the tips. I've had this issue for 2 months and would have logged out and into Flickr many times since then. I have an unpaid version and have only 5 images there. Since Tony Tiwane said he can recreate the issue, I assume I'm not the only one that has it and they will find a solution for it.

Thanks again!
Chris



Chris Moeseneder

Senior Experimental Scientist   |  CSIRO

***@***.******@***.***>  |  07 3833 5946  |  0468 932 499

________________________________

**Comment by Miudod2n** [comment]
Hello,

Sorry to revive this, but I just stumbled upon the same error when trying to upload a custom pic for the taxon Cardus / Cardus crucifer. https://www.inaturalist.org/taxa/96773-Cardus-crucifer

My idea was to add a taxon page photo for this genus because it gets frequently confused with the homonym plant genus Cardus, so maybe people typing the name will see the image of a crab/lobster image and realize something is wrong there. (I'm using an image from a old drawing that is probably in public domain).

I've successfully added a picture on flickr to use as taxon page for a fungus beetle that have been wrongly identified in both inat and flickr for a long time. That image still worked when i pointed it to the Cardus genus too, so the flickr import functionality still works.

I tried uploading the same picture of the crab/lobster again in different formats but the same error appears. I tried to point to Mr. Moeseneder picture and it gives the same error as well.

The browser console messages indicate it is an internal server error (code 500)
![error](https://github.com/inaturalist/inaturalist/assets/29958119/4120563c-cebf-4aba-92a0-0ef172661adf)
